include/ConfigManager.h

```diff
@@ -230,8 +235,27 @@ class EXPORT ConfigManager
 						const QString & value );
 	void deleteValue( const QString & cls, const QString & attribute);
 
-	void loadConfigFile();
-	void saveConfigFile();
+	struct ConfigLoadResult
+	{
+		ConfigLoadResult() :
+			failuresOccurred(false) {}
+
+		ConfigLoadResult(int _errorLine, int _errorColumn,
+				 QString const & _errorString) :
+			failuresOccurred(true),
+			errorLine(_errorLine),
+			errorColumn(_errorColumn),
+			errorString(_errorString)
+		{}
+
+		bool failuresOccurred;
+		int errorLine;
+		int errorColumn;
+		QString errorString;
```

Comment:
Is everyone OK with this naming convention? At the moment, we seem to have two conflicting conventions for struct member variables:

We have the classic `m_varName` (e.g. https://github.com/LMMS/lmms/blob/master/include/MainWindow.h#L187) and also just `varName` (e.g. https://github.com/LMMS/lmms/blob/master/include/Mixer.h#L86).

Arguments can be made for either one. Personally, I've always felt that the `m_` denotes some amount of privacy, and that variables meant to be accessed externally shouldn't have the `m_`. The `m_` convention is really only useful when accessing a variable using an implicit `[this->]m_var` to emphasis where the variable lives.

So I prefer the naming @michaelgregorius has chosen here for Plain-Old-Data types, but we should establish a convention.